src/GridInterpolations.jl

```diff
@@ -318,6 +318,29 @@ function interpolants(grid::SimplexGrid, x::Vector)
     return index::Vector{Int}, weight::Vector{Float64}
 end
 
+# Returns a vector of length num_vertices with the n-dimensional vertices in deterministic order
+function vertices(grid::AbstractGrid)
+
+    vertex_list = Vector{Vector{Float64}}(length(grid))
```

Comment:
Would it be better to make this a matrix (and then do one allocation)?

Reply:
Right you are. In my head it was a list of vectors but of course a matrix makes sense. How does it look now?

Reply:
For future reference, I personally *much* *much* *much* prefer a vector of vectors in terms of style because then it's not ambiguous which axis you should iterate over. Of course there is a cost to allocating a bunch of tiny vectors, but I don't think there is any extra cost if we use `SVector`s http://juliaarrays.github.io/StaticArrays.jl/stable/pages/api.html#Arrays-of-static-arrays-1

Reply:
I think it is pretty standard to store samples as columns in a matrix since Julia is a column-major language. The outer loop would iterate over the columns. Memory allocations can be expensive and it is kind of nice to have it all in one contiguous block. Though, we should not necessarily always sacrifice human interpretability for speed, but I think this is probably easy enough to understand and is consistent with convention.